File: src/main/java/eladkay/quaritum/common/item/misc/ItemPicture.java
package eladkay.quaritum.common.item.misc;

import eladkay.quaritum.common.item.base.ItemMod;
import eladkay.quaritum.common.lib.LibNames;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.Item;
import net.minecraft.item.ItemStack;

import java.util.List;

//This item is here just for its texture.
public class ItemPicture extends ItemMod {
    public ItemPicture() {
        super(LibNames.PICTURE, "picture", "questionMark");
    }

    @Override
    public void getSubItems(Item itemIn, CreativeTabs tab, List<ItemStack> subItems) {
        //noop
    }
}

File: src/main/java/eladkay/quaeritum/common/item/misc/ItemPicture.kt
package eladkay.quaeritum.common.item.misc


import com.teamwizardry.librarianlib.features.base.item.ItemMod
import eladkay.quaeritum.common.lib.LibNames
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack
import net.minecraft.util.NonNullList

//This item is here just for its texture.
class ItemPicture : ItemMod(LibNames.PICTURE, "picture", "questionMark") {

    override fun getSubItems(tab: CreativeTabs, subItems: NonNullList<ItemStack>) {
        // NO-OP
    }
}
